extensions/positron-r/package.json

```diff
@@ -203,6 +203,15 @@
             "default": [],
             "markdownDescription": "%r.configuration.customBinaries.markdownDescription%"
           },
+          "positron.r.customExcludes": {
```

Comment:
Instead of `positron.r.interpreters.exclude`, which would be more consistent with the python settings, I opted for more alignment with the existing R settings `positron.r.customRootFolders` and `positron.r.customBinaries`.

Reply:
Agreed 👍 This has now been updated!